packages/back-end/src/services/organizations.ts

```diff
@@ -156,10 +156,22 @@ export function getRole(
   return getDefaultRole(org);
 }
 
-export function getNumberOfMembersAndInvites(
+export function getNumberOfUniqueMembersAndInvites(
   organization: OrganizationInterface
 ) {
-  return organization.members.length + (organization.invites?.length || 0);
+  // There was a bug that allowed duplicate members in the members array
+  const uniqueMembers = [
+    ...new Map(
+      organization.members.map((member) => [member.id, member])
+    ).values(),
+  ];
+
+  const uniqueInvites = [
+    ...new Map(
+      organization.invites.map((invite) => [invite.email, invite])
+    ).values(),
+  ];
+  return uniqueMembers.length + uniqueInvites.length;
```

Comment:
This works fine, but you can simplify it a bit by using Sets instead of Maps and using `size` instead of converting back to an array.

```ts
const numMembers = new Set(organization.members.map((m) => m.id)).size;
const numInvites = new Set(organization.invites.map((i) => i.email)).size;

return numMembers + numInvites;
```